docs/whats-new.md

```diff
@@ -2,6 +2,109 @@
 
 This page contains highlights of each deck.gl release. Also check our [vis.gl blog](https://medium.com/vis-gl) for news about new releases and features in deck.gl.
 
+## deck.gl v8.1
+
+### Growing Tile Solutions
+
+<table style="border: 0;" align="center">
+  <tbody>
+    <tr>
+      <td>
+        <img style="max-height:200px" src="https://raw.github.com/uber-common/deck.gl-data/master/images/whats-new/tile-2d.gif" />
+        <p><i>Non-geospatial TileLayer</i></p>
+      </td>
+      <td>
+        <img style="max-height:200px" src="https://raw.github.com/uber-common/deck.gl-data/master/images/whats-new/mvt-layer.jpg" />
+        <p><i>MVTLayer</i></p>
+      </td>
+      <td>
+        <img style="max-height:200px" src="https://raw.github.com/uber-common/deck.gl-data/master/images/whats-new/terrain.jpg" />
+        <p><i>TerrainLayer</i></p>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+
+The `@deck.gl/geo-layers` module added many new features to address popular tiled data use cases.
+
+#### TileLayer
+
+Multiple bugs have been fixed in the [TileLayer](/docs/layers/tile-layer.md) regarding data fetching and tree traversal.
+
+The layer now supports non-geospatial views. Check out this [example](https://github.com/uber/deck.gl/tree/master/examples/website/image-tile) by [@ilan-gold](https://github.com/ilan-gold) that renders a 576 Megapixel image of the moon.
+
+New props are added to better control the layer's behavior:
+
+- `maxCacheByteSize`: for precise management of memory usage
+- `refinementStrategy`: to reduce flashing/overlapping during loading
+- `tileSize` (non-geospatial only)
+
+#### MVTTileLayer
+
+Based on the `TileLayer`, [MVTTileLayer](/docs/layers/mvt-layer.md) loads and renders tiles in the [Mapbox Vector Tiles specification](https://github.com/mapbox/vector-tile-spec). This new layer make it easier to leverage the many great [open source tools](https://github.com/mapbox/awesome-vector-tiles) in use with deck.gl.
+
+This effort is led by contributors from [CARTO](https://carto.com/).
+
+#### TerrainLayer
+
+[TerrainLayer](/docs/layers/terrain-layer.md) loads color-encoded heightmap and reconstructs 3D mesh surfaces. Check out our [example](https://github.com/uber/deck.gl/tree/master/examples/website/terrain).
+
+### ESRI + deck.gl
```

Comment:
@damix911

Reply:
Perfect! I wonder if the word _"experimental"_ should be stressed in the API docs too.

I opened a PR for it: #4396. It also fixes a couple of links in the ArcGIS API docs.